specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-02-01/examples/ManagedClustersCreate_Update.json

```diff
@@ -20,6 +20,7 @@
             "vmSize": "Standard_DS1_v2",
             "osType": "Linux",
             "type": "VirtualMachineScaleSets",
+            "scaleDownMode": "Deallocate",
```

Comment:
Why you are updating old example whilst updating new swagger?

Reply:
good catch, fixed